src/Reflection/ReflectionObject.php

```diff
@@ -0,0 +1,465 @@
+<?php
+
+namespace BetterReflection\Reflection;
+
+use BetterReflection\Reflector\ClassReflector;
+use BetterReflection\SourceLocator\EvaledCodeSourceLocator;
+use PhpParser\Node\Stmt\Property as PropertyNode;
+use PhpParser\Builder\Property as PropertyNodeBuilder;
+
+class ReflectionObject extends ReflectionClass
```

Comment:
Added some good old commenting to describe the intent of the existing tests that cover this off